packages/grid/src/__tests__/__e2e__/Grid.cy.tsx

```diff
@@ -606,4 +556,44 @@ describe("Grid", () => {
     checkRowSelected(2, true);
     checkRowSelected(3, true);
   });
+
+  describe("Column Sorting", () => {
+    it.only("should sort column values when sorting is enabled", () => {
+      cy.mount(<RowSelectionModes />);
+
+      // first click: sort in ascending order
+      cy.findAllByTestId("column-header")
+        .eq(2)
+        .should("have.text", "B")
+        .as("sortableHeader")
+        .click()
+        .should("have.attr", "aria-sort", "ascending")
+        .then(() => {
```

Comment:
Was this `then()` necessary?

Reply:
nice one - could do with just `findCell` directly